spec/requests/users_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'users', type: :request do
+  describe 'GET /users' do
+    it 'works! returns correct status code' do
+      get users_path
+      expect(response).to have_http_status(200)
+    end
+
+    it 'works! renders index view correctly' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'works! reponse body includes <h1>This is a list of users<h1> ' do
+      get users_path
+      expect(response.body).to include('<h1>This is a list of users<h1>')
+    end
+  end
+end
+
+RSpec.describe 'users/:id', type: :request do
```

Comment:
- [ ] Here as well, I would recommend removing this `describe` block from here to follow the DRY and rails principle.